src/solvers/smt2_incremental/smt2_incremental_decision_procedure.cpp

```diff
@@ -100,6 +94,46 @@ void smt2_incremental_decision_proceduret::define_array_function(
       convert_expr_to_smt(elements.at(i)))};
     solver_process->send(element_definition);
   }
+}
+
+void smt2_incremental_decision_proceduret::initialize_array_elements(
+  const array_of_exprt &array,
+  const smt_identifier_termt &array_identifier)
+{
+  const smt_sortt index_type =
+    convert_type_to_smt_sort(array.type().index_type());
+  const smt_identifier_termt array_index_identifier = smt_identifier_termt{
```

Comment:
:pick: :thinking: The `smt_identifier_termt` type is duplicated on each side of the `=`. You could use `auto` on the left hand side or initialise without using `=` in order to get rid of the duplication.

Reply:
Removed